### ISSUE TITLE
Bug(Accessibility): Platform Search field does not meet Minimum Contrast Ratio

### ISSUE THREAD

**Issue by ranjithPRao** [SAP/fundamental-ngx#8904]
Is this a bug, enhancement, or feature request?
bug

Briefly describe your proposal.
1. Platform Search field does not meet Minimum Contrast Ratio of 4.5:1
    Currently the ratio is : 3.2:1

<img width="558" alt="image" src="https://user-images.githubusercontent.com/116718610/198003747-d3fb6e99-c14e-4089-8850-3ec960410055.png">

2. Also, the Search icon focus is 3 sided:
<img width="558" alt="image" src="https://user-images.githubusercontent.com/116718610/198255203-dc50f1b7-b86c-4d67-91e2-fdb08ada338f.png">

Which versions of Angular and Fundamental Library for Angular are affected? (If this is a feature request, use current version.)
@fundamental-ngx/core": "0.35.4
@fundamental-ngx/platform": "0.35.4

 If this is a bug, please provide steps for reproducing it.

Please provide relevant source code if applicable.

<fdp-search-field  [disableRefresh]="true" placeholder="Search"  class="supplier-search"
                        (inputChange)="onInputChange($event)"
                            (searchSubmit)="filterInvitedSuppliers($event)"></fdp-search-field>

Is there anything else we should know?

**Comment by platon-rov** [comment]
fixed.
